lib/web_ui/test/keyboard_test.dart

```diff
@@ -241,54 +241,6 @@ void main() {
 
       Keyboard.instance.dispose();
     });
-
-    test('ignores keyboard events triggered on text fields', () {
-      Keyboard.initialize();
-
-      int count = 0;
-      ui.window.onPlatformMessage = (String channel, ByteData data,
-          ui.PlatformMessageResponseCallback callback) {
-        count += 1;
-      };
-
-      useTextEditingElement((html.Element element) {
-        final html.KeyboardEvent event = dispatchKeyboardEvent(
-          'keydown',
-          key: 'SomeKey',
-          code: 'SomeCode',
-          target: element,
-        );
-
-        expect(event.defaultPrevented, isFalse);
-        expect(count, 0);
-      });
-
-      Keyboard.instance.dispose();
-    });
-
-    test('the "Tab" key should never be ignored', () {
```

Comment:
This test should remain. Is it failing after your change (it shouldn't)?

Reply:
OK, I am going to get this test back as you said.